### ISSUE TITLE
Write FE_Discontinuous

### ISSUE THREAD

**Issue by bangerth** [dealii/dealii#6059]
I have a student in my deal.II class who is implementing a "weak Galerkin" method. It's not terribly important what exactly this is, but it uses an auxiliary space for the gradients that is, in essence, a Raviart-Thomas space without any continuity requirements. That is, on *every cell individually* it is an RT space. For the purposes of that program, we can work by just considering one cell at a time.

But in the long run, it would be nice to reconstruct/postprocess the gradient in this space so that it can be output or an error computed. The problem is that we don't have a finite element class for this -- `FE_RaviartThomas` has continuity requirements.

In should not actually be very difficult to write a class, say `FE_Discontinuous`, that takes any element we have as argument and builds a finite element that has locally the same basis, but no continuity between cells.

**Comment by bangerth** [comment]
I've come to believe that `FE_Broken` is a better name for this element -- because it relates to the "broken finite element space" as it is often called.

In reading through some other patches, I've discovered that we have the `FE_DGVector` class that actually might already serve this purpose at least for vector valued cases. But one would have to evaluate whether that is actually true.

**Comment by bangerth** [comment]
Plus there are classes `FE_DGRaviartThomas`, `FE_DGBDM`, and `FE_DGNedelec`. Who knew. I think they correspond to what we need.

Now we just have to update the step-61 tutorial by using them. @sophy1029 -- if you're adventurous!

**Comment by sophy1029** [comment]
@bangerth  Yes! I suppose we can use FE_DGRaviartThomas instead of FE_RaviartThomas to calculate discrete weak gradients? And we can also output Darcy velocity by creating a dof_handler associated with it?

**Comment by bangerth** [comment]
On a single cell, there should be no difference. So you won't *need* it for computing the weak gradient, though it would probably be the more consistent thing to do.

For outputting the Darcy velocity, this indeed seems like the exact right thing to use! It would be great if you wanted to give this a try!

**Comment by sophy1029** [comment]
Yes! I'd like to try to calculate and output the numerical velocity.

**Comment by bangerth** [comment]
Then go right ahead!
